1-estrategia/README.md

```diff
@@ -2,22 +2,129 @@
+
+
 ### Visión
 
+Ser referentes número uno para todos los refugiados e inmigrantes forzados que llegan al País Vasco para que tengan una correcta inserción dentro de la comunidad.
+
+__*¿Cómo veo mi producto?*__
+
+Queremos que Aupa sea un sitio web donde los migrantes puedan encontrar información útil para la acogida en el País Vasco, Además será una plataforma que les permita interactuar y contar sus experiencias al mismo tiempo que se van adaptando al entorno. 
+
+__*¿Cómo quiero que sea el producto?*__
+
+El producto está pensado como una línea de tiempo.
+Nos referimos a una secuencia de pasos que tiene como punto de partida el momento en que los inmigrantes llegan al País Vasco, desde ahí, se da inicio a una serie de pasos que al ser completados, puedan lograr una plena inserción dentro de la comunidad.
+
+![Los pasos](https://github.com/DeustoPWEB2018/proyectoweb-migraciones/blob/85faae59c23aa3f2042f300a18d008d457ecc177/1-estrategia/pasos.png?raw=true)
+
+__*¿Cuál es el propósito de mi producto?*__
+
+Buscamos ser ese facilitador para los inmigrantes forzados que pueda brindarle un marco formal ó camino a seguir para resolver sus necesidades e inquietudes con el menor plazo de tiempo posible.
+De un modo simplificado, partiremos dando por sentado una situación común a todos los inmigrantes forzados la que denominaremos "punto cero" con la siguiente característica: inmigrante forzado que llega al Pais Vasco sin tener resuelto de antemano los papeles migratorios, alojamiento, alimentación, educación, trabajo, atención sanitaria, red de vínculos sociales, etc. Desde allí, existirán "x pasos", que prevén, facilitar el camino a seguir para la resolución ó superación de todas las necesidades que tienen hasta alcanzar el paso final que les permita la plena inserción en la comunidad vasca.
+Desglosar el proceso en pasos permitirá que cada inmigrante forzado pueda partir desde el paso que se adapte a su situación particular y luego si, continuar con los pasos que restan.
+
+__*¿Quiénes son los clientes potenciales asociados con mi producto?*__
+
+El público al que nos dirigimos son migrantes entre 18 y 45 años que llegan al País Vasco. Nos hemos centrado en esta franja de edad, ya que según una encuesta realizada por el Gobierno Vasco en el año 2010 el 47% de los inmigrantes que llegan tienen entre 25 y 44 años, un 34% son menores de 25 años y tan solo el 19% son mayores de 45 años. Por esta razón, hemos concretado nuestro cliente potencial en este rango de edad, ya que además son las personas que más uso hacen de plataformas web, redes sociales y aplicaciones para móvil o tablets. 
```

Comment:
Este estudio tiene que estar referenciado en las fuentes al final del documento.

Tendréis que determinar a lo largo de este documento si las necesidades de un migrante de 19 años y la de uno de 35 son las mismas; si os centraréis en aquellas necesidades que son comunes; o si descartaréis algún segmento de este rango de edad por plantear necesidades diferentes.